### ISSUE TITLE
Question about gradients anomaly during training

### ISSUE THREAD

**Issue by zju-jiyicheng** [WooSunghyeon/dropbp#3]
Excellent work! However, after modifying model.py and trainer.py in a custom version, I found that all gradients became zero during fine-tuning. I also encountered a similar issue when using the HuggingFace version you provided. Could you please let me know what might be causing this?

Looking forward to your reply!
